pkg/kubernetes/deploy.js

```diff
@@ -76,6 +79,48 @@ define([
         $("#deploy-app-dialog").dialog("wait", promise);
     }
 
+    function deploy_nulecule() {
+        var promise = validate()
+            .fail(function(exs) {
+                console.log("validated failed");
+                $("#deploy-app-dialog").dialog("failure", exs);
+            })
+            .done(function(fields) {
```

Comment:
When validation is successful, you should any prior errors with $("#deploy-app-dialog").dialog("failure", null)